frappe/email/doctype/notification/notification.py

```diff
@@ -642,18 +659,16 @@ def get_attachment(self, doc):
 				title=_("Error in Notification"),
 			)
 		else:
-			return [
-				{
-					"print_format_attachment": 1,
-					"doctype": doc.doctype,
-					"name": doc.name,
-					"print_format": self.print_format,
-					"print_letterhead": print_settings.with_letterhead,
-					"lang": frappe.db.get_value("Print Format", self.print_format, "default_print_language")
-					if self.print_format
-					else "en",
-				}
-			]
+			return {
+				"print_format_attachment": 1,
+				"doctype": doc.doctype,
+				"name": doc.name,
+				"print_format": self.print_format,
+				"print_letterhead": print_settings.with_letterhead,
+				"lang": frappe.db.get_value("Print Format", self.print_format, "default_print_language")
```

Comment:
```suggestion
				"lang": doc.get("language") or frappe.db.get_value("Print Format", self.print_format, "default_print_language")
```

Not related to this PR, but using the specific transaction's language with highest priority would be correct. Maybe we can sneak this in here 😁 (or create a separate PR later)